srcpkgs/glibc/template

```diff
@@ -123,23 +123,28 @@ do_build() {
 	env LDFLAGS.so="-Wl,--hash-style=both" LDFLAGS-rtld="-Wl,--hash-style=both" \
 		make ${makejobs}
 }
+
 do_install() {
 	vlicense LICENSES
 	# Create DESTDIR/etc/ld.so.conf
 	mkdir -p ${DESTDIR}/etc
 	echo "include /etc/ld.so.conf.d/*.conf" > ${DESTDIR}/etc/ld.so.conf
 
-	cd build
-	make install_root=${DESTDIR} install
+	( cd build && make install_root=${DESTDIR} install )
+
+	# The C.UTF-8 locale should always be available
+	vmkdir usr/lib/locale
+	I18NPATH=./localedata localedef --prefix="${DESTDIR}" \
```

Comment:
Use the system localedef when cross compiling and the recently built localedef when native compiling.
Also, make sure to pass `--big-endian` or `--little-endian` when cross compiling.

See: https://salsa.debian.org/glibc-team/glibc/-/blob/6c8b05d8a7138593c80908d5fe5b928d2ac038d6/debian/rules.d/build.mk#L340-358